### PR TITLE
Updatereadme

### PR DESCRIPTION
**Pull request by hmunfru**
@jframos

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -32,18 +50,201 @@ of that.
...
+
+The ``test`` target for each module in the PaaS Manager is used for running the unit tests in both components of
+PaaS Manager GE:
+
+Please have a look at the section `building from source code
+<doc/admin-guide.rst#install-paas-from-source>`_ in order to get more
+information about how to prepare the environment to run the
+unit tests.
+
+
```
Comment (flopezag) [inline README.rst]: Please ass how can you execute the unit tests and how to execute a specific unit test.

File: README.rst [hunk 2]
```diff
@@ -32,18 +50,201 @@ of that.
...
+Acceptance tests
+----------------
+
+In the following path you will find a set of tests related to the
+end-to-end funtionalities.
+
+- `PaaS Manager Aceptance Tests <https://github.com/telefonicaid/fiware-paas/tree/develop/test>`_
+
```
Comment (flopezag) [inline README.rst]: Please add how can you execute the acceptance tests and how to prepare the environment if something is needed in order to execute these tests.

File: README.rst [hunk 3]
```diff
@@ -1,16 +1,34 @@
-PaaS Manager - Overview
-____________________________
-
+FI-WARE PaaS Manager
```
Comment (jframos) [inline README.rst]: =========================== 
 FIWARE PaaS Manager | Pegasus
 ===========================

File: README.rst [hunk 4]
```diff
@@ -32,18 +50,201 @@ of that.
...
+
+
+Installation
+------------
+
+Using FIWARE package repository (recommended)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Refer to the documentation of your Linux distribution to set up the URL of the
+repository where FIWARE packages are available (and update cache, if needed):
```
Comment (jframos) [inline README.rst]: Check the use of ``::`` for code blocks in all doc files.

File: README.rst [hunk 5]
```diff
@@ -32,18 +50,201 @@ of that.
...
+(no special privileges required):
+
+::
+
+    $ service fiware-paas status
+
+
+API Overview
+============
+
```
Comment (jframos) [inline README.rst]: You should add here an overview of the main API operations.

File: README.rst [hunk 6]
```diff
@@ -1,16 +1,34 @@
...
-
+FI-WARE PaaS Manager
+====================
 
 | |Build Status| |Coverage Status| |StackOverflow|
 
-What you get
-============
+Introduction
+=======================
```
Comment (jframos) [inline README.rst]: I think the content of this section should be under the main title.

File: README.rst [hunk 7]
```diff
@@ -32,18 +50,201 @@ of that.
...
+We can also force a service restart::
+
+    $ service fiware-paas restart
+
+
+Configuration file
+------------------
+
+The configuration of PaaS Manager is in configuration_properties table in the database.
+There, it is required to configure::
```
Comment (jframos) [inline README.rst]: I think you should show here how to configure the ``paasmanager.xml`` file in order to setup the database connection.

**Comment by jframos** [comment]
You should describe in the **Installation** section how to install PaaSManager on Centos using the automated script located in ``scripts/bootstrap/centos.sh``

File: README.rst (hunk 7 above)
Comment (hmunfru) [inline README.rst]: Done

**Comment by geonexus** [comment]
You should add a index to the different sections at the beginning of this Readme.

.. contents:: :local:

File: README.rst [hunk 8]
```diff
@@ -1,16 +1,33 @@
...
-
+===========================
+FIWARE PaaS Manager | Pegasus
+===========================
 
 | |Build Status| |Coverage Status| |StackOverflow|
 
-What you get
-============
+This is the code repository for FIWARE Pegasus, the reference implementation
```
Comment (geonexus) [inline README.rst]: You should add a Section called Introduction and start it with:

This is the code repository for FIWARE Pegasus...

After adding an index

**Comment by jmmovilla** [comment]
LGTM

**Pull request merged**